### PR TITLE
Added support for AD4696, AD4697, AD4698

### PR DESCRIPTION
**Pull request by SaikiranGudla**
1) Added ad469x.py
2) Added a test script - test_ad469x.py
3) Added an example code - ad469x_example.py
4) Updated index.rst
5) Updated init.py
6) Updated supported_parts.md
7) Added adi.ad469x.rst

Signed-off-by: Gudla <Saikiran.Gudla@analog.com>

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How has this been tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

**Test Configuration**:
* Hardware:
* OS:

# Documentation

If this is a new feature or example please mention or link any documentation. All new hardware interface classes require documentation.

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have signed off all commits and they contain "Signed-off by: <insert name>"
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: adi/ad469x.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+            "ad4698",
+        ]
+
+        self._ctrl = None
+
+        if not device_name:
+            device_name = compatible_parts[0]
+        else:
+            if device_name not in compatible_parts:
+                raise Exception("Not a compatible device: " + device_name)
```
Comment (tfcollins) [inline adi/ad469x.py:66]: Can you extend this message a bit. For example:
```python
raise Exception(f"Not a compatible device: {device_name} . Supported device names are: {', '.join(compatible_parts)}" )
```

Just to help users out

Comment (SaikiranGudla) [reply adi/ad469x.py]: The code has been updated.

**Comment by SaikiranGudla** [comment]
@tfcollins, The code was failing in the Documentation Tests saying Missing ad469x in supported_parts.md file. I'd ad4696,97,98 (of the ad469x family) added in the file. When I tried to add ad469x to the file and run the command on my local machine, it passed. So, do I need to add ad469x along with the ad4696,97,98 parts or remove them?

**Comment by tfcollins** [comment]
The documentation checks work by comparing the supported parts list to the module class names. There are two options:
1. Expose the parts as individual classes like ad936x: https://github.com/analogdevicesinc/pyadi-iio/blob/master/adi/ad936x.py#L238
2. Add ad469x to the doc check ignore list: https://github.com/analogdevicesinc/pyadi-iio/blob/master/tasks.py#L162

You can run the doc check locally with the invoke task checkparts https://analogdevicesinc.github.io/pyadi-iio/dev/index.html#invoke

-Travis

**Pull request merged**